src/main/java/thePackmaster/cardmodifiers/witchesstrikepack/InscribedMod.java

```diff
@@ -0,0 +1,59 @@
+package thePackmaster.cardmodifiers.witchesstrikepack;
+
+import basemod.abstracts.AbstractCardModifier;
+import basemod.helpers.CardModifierManager;
+import com.megacrit.cardcrawl.actions.utility.UseCardAction;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.core.AbstractCreature;
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.helpers.CardHelper;
+import com.megacrit.cardcrawl.localization.CardStrings;
+import com.megacrit.cardcrawl.orbs.AbstractOrb;
+import thePackmaster.SpireAnniversary5Mod;
+import thePackmaster.cards.AbstractPackmasterCard;
+import thePackmaster.orbs.PackmasterOrb;
+
+public class InscribedMod extends AbstractCardModifier {
+    private CardStrings uiStrings =  CardCrawlGame.languagePack.getCardStrings(SpireAnniversary5Mod.makeID("Inscribed"));
+    private boolean inherent = false;
+    private boolean Todesc = true;
+    public InscribedMod(boolean inherent,boolean todesc) {
+        Todesc = todesc;
+    }
+    public void onUpdate(AbstractCard card) {
```

Comment:
I don't think they should have a special glow.

Reply:
noted. this was just something copied from the mod the mechanic came from will remove it.